pkgs/top-level/python2-packages.nix

```
@@ -57,7 +61,7 @@ with self; with super; {
 
   pyparsing = callPackage ../development/python2-modules/pyparsing { };
 
-  pytest = pytest_4;
+  pytest = super.pytest_4;
```

```suggestion
  pytest = self.pytest_4;
```

`pytest_4` wouldn't be defined prior to this overlay

---

Thank you!